tools/packrom/packrom.cpp

```diff
@@ -67,9 +73,10 @@ typedef struct{
 	u8 author[32];
 	u8 icon[16*16];
 	u32 crc32;
-	u8 mouse;
+	u8 psupport; //peripherals supported
 	u8 description[64];
-    u8 reserved[114];
+	u8 pdefault; //peripherals to enable by default(Emulator)
+	u8 reserved[113];
```

Comment:
Can you ensure that the "reserved" array gets initialized to zero before writing to the output file, so any future extensions that need to use some of the reserved bits are guaranteed to have them properly initialized to default values?

Maybe something like `memset(rom.header.reserved, 0, sizeof(rom.header.reserved));`